src/NuGet/Microsoft.Net.Compilers.Toolset/AnyCpu/Microsoft.Net.Compilers.Toolset.Package.csproj

```diff
@@ -62,8 +62,7 @@
       <TfmSpecificPackageFile Include="@(_FileWithPath)" PackagePath="%(_FileWithPath.TargetPath)" />
     </ItemGroup>
 
-    <!-- Used for debugging parts of the toolset (e.g., build tasks).
-      Doesn't work in VS (where `<ProjectReference Targets="Publish" />` doesn't have any effect), hence we set `ContinueOnError="true"`. -->
+    <!-- Used for debugging parts of the toolset (e.g., build tasks) via `dotnet build -p:InnerTargets=GetFiles`. -->
     <Copy SourceFiles="@(_FileWithPath)"
           DestinationFiles="$(BaseOutputPath)$(Configuration)\%(_FileWithPath.TargetPath)"
           SkipUnchangedFiles="true" />
```

Comment:
What might be cleaner would be to move this copy to a separate target that depends on _GetFiles, and then point to that target in your VS Code path. Otherwise you might still be producing files that nobody is looking at.